projects/packages/publicize/src/class-publicize-base.php

```diff
@@ -1475,6 +1476,26 @@ public function publicize_checkbox_default( $checked, $post_id ) {
 		return $checked;
 	}
 
+	/**
+	 * Adds the sig image to the meta tags array.
+	 *
+	 * @param array $tags Current tags.
+	 */
+	public function get_sig_image_for_post( $tags ) {
+		$post_settings = new Social_Image_Generator\Post_Settings( get_the_ID() );
+		if ( $post_settings->is_enabled() ) {
+			$tags = array_merge(
+				$tags,
+				array(
+					'og:image'        => $post_settings->get_image_url( get_the_ID() ),
```

Comment:
We need to use the utility function instead: https://github.com/Automattic/jetpack/blob/trunk/projects/packages/publicize/src/social-image-generator/utilities.php#L18

The utility function returns the actual generated URL. The `Post_Settings()->get_image_url()` just returns the background image for the generated image.